daemon/bindata.go

```diff
@@ -1,5 +1,9 @@
 // Code generated by go-bindata.
 // sources:
+// ../bpf/.bpf_lxc.c.swn
```

Comment:
This would need to go into daemon/Makefile as well which indicates what to ignore:
```
GO_BINDATA := go-bindata -prefix ../ -mode 0640 -modtime 1450269211 \
	-ignore Makefile -ignore bpf_features.h -ignore lxc_config.h \
	-ignore netdev_config.h -ignore node_config.h -ignore '.+\.o$$' \
	-ignore '.+\.orig$$' -ignore '.+~$$' -ignore '\.DS_Store'
```